client/sw.js

```diff
@@ -1,7 +1,6 @@
 self.addEventListener('push', (e) => {
   const data = e.data.json();
-  self.registration.showNotification(data.title, {
-    body: data.body,
-    icon: data.icon,
```

Comment:
Why did you remove these options?

Reply:
I debug:  data.boy add data.icon is null